terraform/account/region/data_sources.tf

```diff
@@ -2,3 +2,8 @@ data "aws_kms_alias" "cloudwatch_application_logs_encryption" {
   name     = var.cloudwatch_log_group_kms_key_alias
   provider = aws.region
 }
+
+# data "aws_kms_alias" "s3_encryption" {
+#   name     = var.s3_kms_key_alias
+#   provider = aws.region
+# }
```

Comment:
Are we saving this for a future ticket?

Reply:
Yeah, the next ticket will use this to encrypt the access logs bucket